CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+# Contributing to api-designer
+
+Do you want to help improving this project?? Here are some instructions to get you started. They are probably not perfect, please let us know if anything feels wrong or incomplete.
+
+## Reporting Issues
+
+When reporting [issues](https://github.com/mulesoft/api-designer/issues)
+on GitHub please include the version of api-designer you are using, as well as the name and version of your browser, and your OS. Please also include the steps required to reproduce the problem if possible and applicable. This information will help us review and fix your issue faster.
+
+## Build Environment
+
+Install global tools
+
+```
+npm install -g grunt-cli
+npm install -g bower
+npm install -g karma # Optional for running the test suite
+```
+
+Install node modules
+
+```
+npm install
+```
+
+Install bower modules
+
+```
+bower install
+```
+
+Install webdriver required to run `localScenario` task
+
+```
+node_modules/grunt-protractor-runner/node_modules/protractor/bin/webdriver-manager update
+```
+
+On some systems you need add `node` in front of the line above
+
+```
+node node_modules/grunt-protractor-runner/node_modules/protractor/bin/webdriver-manager update
+```
+
+Run the application locally
+
+```
+grunt server
+```
+
+Run the test suite
+
+```
+grunt test
+```
+
+Build the application
+
+```
+grunt
+```
+
+## Workflow
+
+* we are using [git triangular workflow](http://www.mulesoft.org/legal/contributor-agreement.html)
+* no one, not even the api-designer maintainers, push contributions directly
```

Comment:
Every `*` will be rendering as a bullet in Markdown, but this looks like the same sentence flowing over multiple bullets?